src/pages/Detail/Comment.jsx

```diff
@@ -0,0 +1,151 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '@/components/Button';
+import supabase from '@/supabase/supabaseClient';
+
+const userComment = {
+  img: 'src/assets/images/common/user.png',
+  name: 'mari'
+};
+
+const Comment = () => {
+  const [content, setContent] = useState('');
+  const [comment, setComment] = useState([]);
+
+  const onContent = (e) => {
+    setContent(e.target.value);
+  };
+  useEffect(() => {
+    const fetchData = async () => {
+      const { data, error } = await supabase.from('comments').select('*');
+      if (error) {
+        console.log('error', error);
```

Comment:
```suggestion
      if (error) {
        alert('시스템 오류로 댓글을 가져오지 못했어요, 다시 불러오려면 새로고침해주세요')
        return;
```

에러가 나는 경우에 고객은 아무것도 눈치를 채지 못합니다! 이렇게 쓰는 것은 어떨까요?
(return문 포함)